api.md

```diff
@@ -170,6 +172,63 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
 SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_exists => true);
 ```
 
+---
+## attach_data_node() [](attach_data_node)
+
+Attach a data node to a hypertable. The data node should have been
+previously created using [`add_data_node`](#add_data_node).
+
+When a distributed hypertable is created it will by default use all
+available data nodes for the hypertable, but if a data node is added
+*after* a hypertable is created, the data node will not automatically
+be used by existing distributed hypertables.
+
+If you want a hypertable to use a data node that was created later,
+you must attach the data node to the hypertable using this
+function.
+
+>:WARNING: This function *MUST* be run on an access node.
```

Comment:
It seems like this should be pretty obvious, but it we want this warning we should probably add it to all distributed commands (attach..., detach..., delete_data_node, create_distributed_hypertable...add_data_node is a bit of a special case in that we just can't run it on an existing data node).

Reply:
Hmmm... I almost think in that case we should consider just flagging with a different flag -> like :clustering: or something. I think for now since the other commands aren't fully merged yet, I'm gonna remove this WARNING, and consider a better approach in the next round of docs edits.